go/libraries/doltcore/sqle/reflog_table_function.go

```diff
@@ -199,19 +213,17 @@ func (rltf *ReflogTableFunction) Schema() sql.Schema {
 }
 
 func (rltf *ReflogTableFunction) Resolved() bool {
-	if rltf.refExpr != nil {
-		return rltf.refExpr.Resolved()
+	for _, expr := range rltf.refAndArgExprs {
+		return expr.Resolved()
```

Comment:
You don't want to always return on the first expression – you need to check that all the expressions are resolved. 

You probably want something like `if expr.Resolved() == false { return false }` instead.